.env

```diff
@@ -29,7 +30,26 @@ OKP4_COOKIE_POLICY_URL='https://okp4.network/cookie-policy'
 OKP4_GITHUB_OKP4D_URL='https://github.com/okp4/okp4d'
 OKP4_GITHUB_CONTRACTS_URL='https://github.com/okp4/contracts'
 OKP4_GITHUB_AWESOME_URL='https://github.com/okp4/awesome'
+OKP4_GITHUB_OKP4D_RELEASES_URL='https://github.com/okp4/contracts/releases/tag/v2.0.0'
+OKP4_GITHUB_OBJECTARIUM_URL='https://github.com/okp4/contracts/tree/v2.0.0/contracts/okp4-objectarium'
+OKP4_GITHUB_COGNITARIUM_MESSAGES_URL='https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/src/msg.rs'
+OKP4_GITHUB_LAW_STONE_URL='https://github.com/okp4/contracts/tree/main/contracts/okp4-law-stone/examples/single-source'
+OKP4_GITHUB_PREDICATE_BLOCK_URL='https://github.com/okp4/okp4d/blob/main/x/logic/predicate/block.go'
+OKP4_GITHUB_PREDICATE_ADDRESS_URL='https://github.com/okp4/okp4d/blob/main/x/logic/predicate/address.go'
+OKP4_GITHUB_LOGIC_MODULE_PREDICATES_URL='https://github.com/okp4/okp4d/tree/main/x/logic/predicate'
+OKP4_GITHUB_PROLOG_TEMPLATE_URL='https://github.com/okp4/contracts/tree/main/contracts/okp4-law-stone/examples/multiple-sources'
+OKP4_GITHUB_ONTOLOGY_RHIZOME_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl'
+OKP4_OBJECTARIUM_ADDRESS='T1_OBJECTARIUM_ADDR'
+OKP4_OBJECTARIUM_ID='T1_OBJECTARIUM_ID'
+OKP4_SMART_CONTRACT_LAW_STONE_CODE_ID='T2_LAW_STONE_CODE_ID'
+OKP4_SMART_CONTRACT_COGNITARIUM_CODE_ID='T5_COGNITARIUM_CODE_ID'
```

Comment:
@houssemmimoun27 

Are you sure about these variables?

@antho31 can you confirm ?

Reply:
> @houssemmimoun27
> 
> Are you sure about these variables?
> 
> @antho31 can you confirm ?


I specified these unknown values as env variables but they might never change so it can only be text replacement. As soon as smart contracts are newly deployed, I give you the value of these.